common/lib/aws_client.ts

```diff
@@ -149,4 +149,6 @@ export abstract class AwsClient extends EventEmitter {
   getPluginInstance<T>(iface: any): T {
     return this.pluginManager.getPluginInstance(iface);
   }
+
+  abstract setKeepAlive(keepAlive: boolean, options?: any): Promise<void>;
```

Comment:
Why is setKeepAlive a method on the `awsClient` instead of a connection property like the network timeouts?